listenbrainz/webserver/views/test/test_player.py

```diff
@@ -1,14 +1,9 @@
 from flask import url_for
 
-from listenbrainz.db.testing import DatabaseTestCase
-from listenbrainz.webserver.testing import ServerTestCase
+from listenbrainz.tests.integration import IntegrationTestCase
 
 
-class PlayerViewsTestCase(ServerTestCase, DatabaseTestCase):
-
-    def setUp(self):
-        ServerTestCase.setUp(self)
-        DatabaseTestCase.setUp(self)
+class PlayerViewsTestCase(IntegrationTestCase):
```

Comment:
I see that there are 3 other subclasses of `IntegrationTestCase` in the `listenbrainz.webserver` package. I would have thought that they'd all be in `listenbrainz.tests.integration` Do you think that we should move these 5-6 test files to the integration package, or leave them for now with the understanding that we're going to be merging them into a single test runner in the short term future anyway?

Reply:
Leaving as is makes sense. Further, the change to combine integration and unit tests has been merged now.